e2e/kube/storage/examples_storage_test.go

```diff
@@ -9,18 +11,29 @@ import (
 
 var _ = Describe("Examples", func() {
 
-	Describe("when examples are specified in the docs", func() {
+	Describe("when storage related examples are specified in the docs", func() {
 
+		var (
+			kubectlHelper *testing.Kubectl
+		)
+		kubectlHelper = testing.NewKubectl()
 		const examplesDir = "../../../docs/examples/"
 
-		Context("all examples with storage must be working", func() {
+		Context("all storage related examples with storage must be working", func() {
 
 			It("bosh-deployment with a persistent disk example must work", func() {
 				yamlFilePath := examplesDir + "bosh-deployment/boshdeployment-with-persistent-disk.yaml"
 
+				By("Creating a secret for implicit variable")
+				class, ok := os.LookupEnv("OPERATOR_TEST_STORAGE_CLASS")
+				Expect(ok).To(Equal(true))
+				literalValues := "--from-literal=value=" + class
+				err := kubectlHelper.CreateSecretFromLiteral(namespace, "nats-deployment.var-implicit-operator-test-storage-class", "generic", literalValues)
+				Expect(err).ToNot(HaveOccurred())
+
 				By("Creating bosh deployment")
 				kubectlHelper := testing.NewKubectl()
```

Comment:
I think you meant to extract this into a context variable in https://github.com/cloudfoundry-incubator/cf-operator/pull/390/commits/fcb4e0dc002dcec551f94a2fc157c408095aff56#diff-fdb6df78666e4d68c918c8133e965eb7R17, so this line can go away.

Reply:
Done.